react-native-libraries.json

```diff
@@ -9685,5 +9685,14 @@
     "ios": true,
     "android": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/jakex7/react-native-localization-settings",
+    "examples": [
+      "https://github.com/jakex7/react-native-localization-settings",
```

Comment:
Hello @jakex7, thank you for adding new entry to the directory! 👍 

LGTM, just have one nit:
```suggestion
```

Reply:
Done 😄